car_showroom-2/src/App.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+<navbar />
+<GalleryCard :cars="cars" v-on:emitAlert="emitPrice"></GalleryCard>
+</template>
+
+<script>
+import GalleryCard from "./components/GalleryCard.vue";
+import navbar from "./components/Navbar.vue";
```

Comment:
this is requirement in Practical : **Create a one global component navbar and use it in app.vue.**

declare navbar component globally